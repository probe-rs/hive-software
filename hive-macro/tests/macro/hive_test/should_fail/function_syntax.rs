#[hive_macro::hive_test]
enum Enum {}

fn main() {}
