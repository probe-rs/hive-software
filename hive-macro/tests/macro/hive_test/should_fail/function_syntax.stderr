error: Failed to parse testfunction
 --> tests/macro/hive_test/should_fail/function_syntax.rs:2:1
  |
2 | enum Enum {}
  | ^^^^
  |
  = help: This macro can only be applied to a Hive testfunction, see the example below.
  = note: example: fn my_fancy_test(test_channel: &mut dyn TestChannelHandle, session: &mut Session, target_info: &HiveTargetInfo, defines: &DefineRegistry);
